### PR TITLE
Adding padding to the links in the List Block utility

### PR DESCRIPTION
**Pull request by chris-pearce**

### CODE REVIEW

**Comment by kllevin** [comment]
LGTM

File: utilities/_u-list-block.scss [hunk 1]
```diff
@@ -93,16 +101,20 @@ $u-list-block-border-thickness:     1 !default;
...
   > a {@include to-rem(margin, -$u-list-block-item-padding-small);}
 }
 
 @include generate-at-breakpoints('.u-list-block--small{bp} > li',
   $u-list-block-breakpoints) {
-    @include to-rem(padding, $u-list-block-item-padding-small);
 
-    > a {@include to-rem(margin, -$u-list-block-item-padding-small);}
+  &,
+  > a {@include to-rem(padding, $u-list-block-item-padding-small);}
```
Comment (smgcode) [inline utilities/_u-list-block.scss:115]: Indentation is different than https://github.com/westfieldlabs/scally/blob/u-list-block-update/utilities/_u-list-block.scss#L88-L95

File: utilities/_u-list-block.scss [hunk 2]
```diff
@@ -112,16 +124,20 @@ $u-list-block-border-thickness:     1 !default;
...
 
   > a {@include to-rem(margin, -$u-list-block-item-padding-large);}
 }
 
 @include generate-at-breakpoints('.u-list-block--large{bp} > li',
   $u-list-block-breakpoints) {
-    @include to-rem(padding, $u-list-block-item-padding-large);
 
-    > a {@include to-rem(margin, -$u-list-block-item-padding-large);}
+  &,
```
Comment (smgcode) [inline utilities/_u-list-block.scss:137]: Indentation is different than https://github.com/westfieldlabs/scally/blob/u-list-block-update/utilities/_u-list-block.scss#L88-L95

File: utilities/_u-list-block.scss [hunk 3]
```diff
@@ -131,16 +147,20 @@ $u-list-block-border-thickness:     1 !default;
...
 
   > a {@include to-rem(margin, -$u-list-block-item-padding-huge);}
 }
 
 @include generate-at-breakpoints('.u-list-block--huge{bp} > li',
   $u-list-block-breakpoints) {
-    @include to-rem(padding, $u-list-block-item-padding-huge);
 
-    > a {@include to-rem(margin, -$u-list-block-item-padding-huge);}
+  &,
```
Comment (smgcode) [inline utilities/_u-list-block.scss:160]: Indentation is different than https://github.com/westfieldlabs/scally/blob/u-list-block-update/utilities/_u-list-block.scss#L88-L95

**Pull request merged**